sustainml_cpp/include/sustainml_cpp/nodes/MLModelMetadataNode.hpp

```diff
@@ -36,35 +36,35 @@ namespace utils {
 template<class T> class SamplePool;
 } // namespace utils
 
-namespace ml_task_encoding_module {
+namespace ml_model_metadata_module {
```

Comment:
We may consider using the same namespace as for the MLModelNode, the `::ml_model_provider` as both of them are related